jenkins/promotion/tar-promotion.jenkinsfile

```diff
@@ -0,0 +1,60 @@
+lib = library(identifier: "jenkins@20211123", retriever: legacySCM(scm))
+
+pipeline {
+    options {
+        timeout(time: 1, unit: 'HOURS')
+    }
+    agent none
+    parameters {
+        choice(
+            choices: ['distribution-build-opensearch', 'distribution-build-opensearch-dashboards'],
+            name: 'DISTRIBUTION_JOB_NAME',
+            description: 'What platform is this distribution build for?.'
+        )
+        string(
+            defaultValue: '98',
```

Comment:
Let's not keep any default values, let it fail rather than copying some wrong files and pushing to prod

Reply:
fixed.